pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -370,6 +370,13 @@ func (p *Provider) cachedEvictedFunc(ctx context.Context) func(string, interface
 		}
 		launchTemplate := lt.(*ec2.LaunchTemplate)
 		if _, err := p.ec2api.DeleteLaunchTemplate(&ec2.DeleteLaunchTemplateInput{LaunchTemplateId: launchTemplate.LaunchTemplateId}); err != nil {
+			if awserrors.IsLaunchTemplateNotFound(err) {
+				logging.FromContext(ctx).With(
```

Comment:
No need to log this. Just ignore the error and exit. We have already logged somewhere that we initiated the delete of this launch template already, so adding an additional log to say that we tried again and didn't find it won't really matter to users.

Reply:
If you wanted to, you could directly just call `if awserrors.IgnoreNotFound(err) != nil` as a nice K8s-style syntax for handling this occurcence